api/src/main/resources/liquibase.xml

```diff
@@ -211,10 +211,10 @@
 		</sql>
 	</changeSet>
 
-	<changeSet id="ugandaemr-14032019-0856" author="dbaluku">
-		<comment>Setting  modules to be UgandaEMR Fingerprint,FHIR,Data Entry Statistics to be started manually</comment>
+	<changeSet id="ugandaemr-01042019-2012" author="dbaluku">
```

Comment:
Add check to ensure that what is being changed exists.

Reply:
@slubwama  the "in " sql key word would take of the exception in a case where  property does not  exist